libs/android_defaults.sh

```diff
@@ -1,5 +1,12 @@
 #!/usr/bin/env bash
 
+# Find the NDK.
+pushd ..
+NDK_VERSION=$(./gradlew -q printNdkVersion | tail -1)
```

Comment:
I'm surprised you still need these at all! (But np if you do)

Reply:
Yeah the PATH to the NDK is needed to compile NSS/SQLCipher and generate Android symbols.